tests/benchdnn/conv/conv.cpp

```diff
@@ -667,6 +667,32 @@ void check_known_skipped_case(const prb_t *prb, res_t *res) {
     if (prb->alg == WINO) {
         if (is_cpu()) {
 #if DNNL_CPU_RUNTIME != DNNL_RUNTIME_NONE
+#if DNNL_AARCH64
+            bool pad_ok_f32 = prb->pw <= 1 && prb->ph <= 1 && prb->pw_r <= 1
+                    && prb->ph_r <= 1;
+            bool shape_ok = prb->ndims == 4 && prb->g == 1 && prb->kh == 3
+                    && prb->kw == 3 && prb->sh == 1 && prb->sw == 1
+                    && prb->dh == 0 && prb->dw == 0 && !prb->has_groups
+                    && pad_ok_f32;
+
+            const auto stag = normalize_tag(prb->stag, prb->ndims);
+            const bool stag_is_valid = stag == "any";
+            const auto dtag = normalize_tag(prb->dtag, prb->ndims);
+            const bool dtag_is_valid = dtag == "any";
+            const bool format_ok = (stag_is_valid || dtag_is_valid);
+
+            const bool dir_ok = (prb->dir == FWD_I || prb->dir == FWD_B);
+
+            const auto &po = prb->attr.post_ops;
+            const auto sum_idx = po.find(attr_t::post_ops_t::kind_t::SUM);
+            const bool sum_post_op_ok
+                    = sum_idx == -1 || po.entry[sum_idx].sum.scale == 1.f;
```

Comment:
Just in case, this restricts sum post-op only, it doesn't restrict any other post-ops like eltwise, binary, prelu, etc. Is it intended to restrict sum only?

Reply:
I've changed this to mirror the checks done in acl_winograd_convolution.hpp.